kola/tests/kubeadm/templates.go

```diff
@@ -96,7 +96,13 @@ storage:
       mode: 0755
       contents:
         remote:
-          url: "data:text/plain;base64,{{ .WorkerScript }}"`
+          url: "data:text/plain;base64,{{ .WorkerScript }}"
+    - path: /etc/docker/daemon.json
+      filesystem: root
+      mode: 0644
+      contents:
+        remote:
+          url: data:text/plain;base64,eyJsb2ctZHJpdmVyIjogImpvdXJuYWxkIn0K`
```

Comment:
I think `inline: |` should work, too.
Another alternative is `data:,…` with URL encoding.
The base64 encoding is not readable for humans^^

Reply:
Yeah I wanted to avoid URL encoding - let's give a try to `inline: |`.